tests/common/test_utils.py

```diff
@@ -42,7 +43,7 @@ def initialize_log(log_level='INFO'):
     # eval(__logger__+".log_level"+"("+"#>> Test_utils: [LOG] Initialized"+")", log_level)
 
 
-@staticmethod
+#@staticmethod
```

Comment:
NTC, static method is not callable from outer functions and it has to be commented in these cases